[KOGITO-8369] Provide an API for tests to run Kaniko without Kubernetes/minikube runtime (#17)
kiegroup/container-builder: README.md

Problems on test with SELinux
`sudo setenforce Permissive`

NOTE on Registry container
TO enable the images deletion you need to set the following environment variable:
```bash
REGISTRY_STORAGE_DELETE_ENABLED=true
```
otherwise it will return an HTTP 405 error (Not Allowed).

## Kaniko Vanilla
Kaniko Vanilla is our API to run a Kaniko build outside Kubernetes Cluster

> when is neede to measure the time of a dockerfile to correctly improve the operations.

Reviewer: ```suggestion
when it is needed to measure the time of a dockerfile to correctly improve the operations.
```

Author: fixed

To run Kaniko locally first we need to start a local registry:

```sh 
docker run -d -p 5000:5000 --name registry registry:latest
```

then run a build with

```sh 
docker run \
 --net=host  -v /<projectpath>/examples:/workspace \         
 -v /home/<user>/.docker/config.json:/root/.docker/config.json \         
 -e DOCKER_CONFIG=/root/.docker \         
 gcr.io/kaniko-project/executor:latest \        
 -f /workspace/dockerfiles/Kogito_swf.dockerfile \         
 -d localhost:5000/kaniko-test/kaniko-dockerfile_test_swf \        

>  --force -c /workspace --verbosity debug

Reviewer: ```suggestion
docker run \
  --force \
  -c /workspace \
  --verbosity debug \
  --net=host  \
  -v /<projectpath>/examples:/workspace \         
  -v /home/<user>/.docker/config.json:/root/.docker/config.json \
  -e DOCKER_CONFIG=/root/.docker \
  -f /workspace/dockerfiles/Kogito_swf.dockerfile \         
  -d localhost:5000/kaniko-test/kaniko-dockerfile_test_swf \        
   gcr.io/kaniko-project/executor:latest
```

Reviewer: This will help to read the whole command correctly

Author: fixed

```

> 

Reviewer: maybe add a NOTE that `<user>` needs to be replaced with current user.

Author: fixed

NOTE on Registry container 

> TO enable the delete of the images you need to set then env var `REGISTRY_STORAGE_DELETE_ENABLED=true`

Reviewer 2: ```suggestion
TO enable the images deletion you need to set the following environment variable:
```bash
REGISTRY_STORAGE_DELETE_ENABLED=true
```

```

Author: ok

> otherwise the delete image return a 405 method not allowed

Reviewer 2: ```suggestion
otherwise it will return an HTTP 405 error (Not Allowed).
```

Author: ok

## Kaniko Vanilla

> Kaniko Vanilla is our API to run a Kaniko build outside Kubernetes/minikube/kind

Reviewer 2: ```suggestion
Kaniko Vanilla is our API to run a Kaniko build outside Kubernetes Cluster
```

Author: ok

when is neede to measure the time of a dockerfile to correctly improve the operations.

> If you need to run Kaniko from a cli you need to start a local registry with 

Reviewer 2: ```suggestion
To run Kaniko locally first we need to start a local registry:
```

Author: ok
